gui/wxpython/nviz/tools.py

```diff
@@ -741,14 +741,14 @@ def _createDataPage(self):
                 style=fpb.FPB_DEFAULT_STYLE,
                 extraStyle=fpb.FPB_SINGLE_FOLD,
             )
-        except:
+        except (AttributeError, TypeError):
```

Comment:
I have removed try except to just put the latest FoldPanelBar, however I changed the definiton of `style` from `FPB_DEFAULT_STYLE` to `FPB_SINGLE_FOLD` since earlier wasn't clearly defined in the docs